tests/car_file_test.rs

```diff
@@ -1,12 +1,20 @@
 use futures::TryStreamExt;
 use iroh_car::*;
-use tokio::fs::{self, File};
 use tokio::io::BufReader;
 
-#[tokio::test]
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen_test::wasm_bindgen_test;
+
+#[cfg(target_arch = "wasm32")]
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+const TEST_V1_CAR: &[u8] = include_bytes!("testv1.car");
```

Comment:
Please add a comment why these are using `include_bytes` instead of reading from disk at runtime